packages/formation/sass/base/_va.scss

```diff
@@ -310,54 +287,15 @@ article > h1 {
   margin-top: 0;
 }
 
-.text-center {
```

Comment:
Is this class still used here? 
https://github.com/department-of-veterans-affairs/vets-website/blob/30c7ce30c852c95d59efe35b17d748859e91a85f/src/applications/dhp-connected-devices/components/PageNotFound.jsx#L9